mu/modes/circuitpython.py

```diff
@@ -24,46 +24,24 @@
 from mu.interface.panes import CHARTS
 
 
-class AdafruitMode(MicroPythonMode):
+class CircuitPythonMode(MicroPythonMode):
     """
-    Represents the functionality required by the Adafruit mode.
+    Represents the functionality required by the CircuitPython mode.
     """
 
-    name = _('Adafruit CircuitPython')
-    description = _("Use CircuitPython on Adafruit's line of boards.")
-    icon = 'adafruit'
-    save_timeout = 0  #: Don't autosave on Adafruit boards. Casues a restart.
-    connected = True  #: is the Adafruit board connected.
+    name = _('CircuitPython')
+    description = _("Boards running Circuit Python.")
+    icon = 'circuitpython'
```

Comment:
Have you checked if the adafruit.png is used anywhere else?
If not then we should probably remove it with the PR as well.